src/mame/drivers/kinst.cpp

```diff
@@ -227,6 +227,8 @@ class kinst_state : public driver_device
 	uint32_t *m_video_base;
 	const uint8_t *m_control_map;
 	emu_timer *m_irq0_stop_timer;
+	
+	uint8_t prot_sel, prot_cnt, prot_rega, prot_regb;
```

Comment:
Please follow the coding style of the class and use the `m_` prefix for instance data members.  Also, remember to register for save states in `machine_start`.